packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -157,6 +157,7 @@ cp -pa .%{gem_dir}/* \
 %{foreman_assets_plugin}
 %{foreman_webpack_plugin}
 %{foreman_webpack_foreman}
+%exclude %{gem_instdir}/public/webpack/foreman-vendor.*
```

Comment:
Should this be part of the precompile macro?
https://github.com/theforeman/foreman-packaging/blob/2224d7916e19f18f1d81fe5c10fc3f61db2e302e/packages/foreman/foreman/foreman.spec#L901-L938
We already remove `*.js.map` there. Probably also good to see if those files still contain the BUILDROOT (which is why we dropped them).

Reply:
Yeah, I also thought of that and agree that this is probably the best place we can have for this cleanup.

Reply:
https://github.com/theforeman/foreman-packaging/pull/3933

Reply:
#3933 got merged and I've updated this PR to not include the exclude (heh).